spec/board_transformer_spec.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../lib/board_transformer'
+
+describe 'Board Transformer' do
```

Comment:
I am not sure if `BoardTransformer` needs to be its own class. Maybe just include these functions in `Board`?